docs/topics/Gateway.md

```diff
@@ -699,6 +728,25 @@ A user's presence is their current state on a guild. This event is sent when a u
 }
 ```
 
+With Rich Presence
```

Comment:
Let's do an example of what Rocket League may actually look like, for consistency with the other example + to make it look official:

```json
{
  "name": "Rocket League",
  "type": 0,
  "application_id": "379286085710381999",
  "state": "In a Match",
  "details": "Ranked Duos: 2-1,
  "timestamps": {
    "start": 1511200066
  },
  "party": {
    "id": "35137100553872123",
    "size": [2, 2]
  },
  "assets": {
    "large_image": "351371005538729000",
    "large_text": "DFH Stadium",
    "small_image": "351371005538729111",
    "small_text": "Silver III"
  }
}
```